azure-auth-helper/src/main/java/com/microsoft/azure/auth/WebAppServer.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.auth;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.apache.commons.lang3.StringUtils;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.net.InetSocketAddress;
+import java.util.Map;
+
+@SuppressWarnings("restriction")
+public class WebAppServer {
+    private static final String ERROR = "error";
+    private static final String CODE = "code";
+    // TODO: need to update this URL after we post the login quick start at
+    // docs.microsoft.com
+    private static final String LOGIN_DOC_URL = "https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme/";
+    private static final String LOGIN_SUCCESS_BODY = "\n" +
```

Comment:
The html body can be managed in a separate .html file as a resource.

Reply:
Done